exercises/killer-sudoku-helper/canonical-data.json

```
@@ -0,0 +1,179 @@
+{
+  "exercise": "killer-sudoku-helper",
+  "cases": [
+    {
+      "description": "Trivial 1-digit cages",
+      "cases": [
+        {
+          "uuid": "2aaa8f13-11b5-4054-b95c-a906e4d79fb6",
+          "description": "1",
+          "property": "combinations",
+          "input": {
+            "cage": {
+              "sum": 1,
+              "size": 1,
+              "exclude": []
+            }
+          },
+          "expected": [[1]]
+        },
+        {
+          "uuid": "4645da19-9fdd-4087-a910-a6ed66823563",
+          "description": "2",
+          "property": "combinations",
+          "input": {
+            "cage": {
+              "sum": 2,
+              "size": 2,
```

if I've understood correctly that `size` refers to the number of digits, shouldn't the size be 1 here? you couldn't have two digits that add up to 2 because 1+1 is not allowed.

```suggestion
              "size": 1,
```

If I have misunderstood, then what does `size` in fact mean?

---

I had the same thought

---

You're right, it's a copy and paste error.